apis/workloads/v1alpha1/replicatedstatemachine_types.go

```diff
@@ -54,28 +74,44 @@ type ReplicatedStateMachineSpec struct {
 	// +optional
 	VolumeClaimTemplates []corev1.PersistentVolumeClaim `json:"volumeClaimTemplates,omitempty"`
```

Comment:
Does the resources support updates?

Btw, there are typos in the comments above.

Reply:
as mentioned above, `VolumeClaimTemplates` is copied from sts and has same behavior with `sts.spec.VolumeClaimTemplates`.